internal/datanode/buffer.go

```diff
@@ -57,10 +264,11 @@ func (bd *BufferData) updateTimeRange(tr TimeRange) {
 type DelDataBuf struct {
 	datapb.Binlog
 	delData *DeleteData
+	item    *Item
 }
```

Comment:
as entriesNum inside DelDataBuf will only increase and never decrease, I modify the function name here to 'accumulate'